src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20161115_pending_workspace_access.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <!--no foreign keys yet-->
+    <changeSet logicalFilePath="dummy" author="mbemis" id="pending-ws-access-1">
+        <sql stripComments="true">
+            create table PENDING_WORKSPACE_ACCESS (
+                workspace_id binary(16) not null,
+                user_email varchar(254) not null,
+                origin_subject_id varchar(254) not null,
+                invite_date timestamp not null,
+                access_level varchar(254) not null,
+                primary key PK_PENDING_WORKSPACE_ACCESS (workspace_id, user_email),
+                foreign key FK_PENDING_WS_ACCESS_WORKSPACE(workspace_id) references WORKSPACE(id),
+                foreign key FK_PENDING_WS_ACCESS_ORIGIN_USER(origin_subject_id) references USER(SUBJECT_ID)
```

Comment:
note to self: fix this create statement so mysql doesn't ignore the foreign key names in favor of auto-generating them